readme.md

```diff
@@ -1,14 +1,34 @@
-# Search plugin for [Refinery CMS](http://www.refinerycms.com)
+# Refinery CMS Search [![Build Status](https://travis-ci.org/refinery/refinerycms-search.svg?branch=master)](https://travis-ci.org/refinery/refinerycms-search)
+
+Simple search engine for [Refinery CMS](http://refinerycms.com). By default, it supports search in `Refinery::Page` and `Refinery::Blog::Post` (if installed), it can be easily configured for another Rails models.
```

Comment:
Grammar:

```
Simple search engine for [Refinery CMS](http://refinerycms.com). By default it supports search in `Refinery::Page`, and `Refinery::Blog::Post` (if installed), but it can be easily configured for other Rails models too.
```